doc/specifications/translations/removal.md

```diff
@@ -14,7 +14,7 @@ the existing Versions of a Content Object and is specified as:
  *       a warning before performing it.
  *
  * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
- *         is the only one Content Object has.
+ *         is the only one Content Object has or it is the main translation.
```

Comment:
A couple of questions on this (ref: https://jira.ez.no/browse/EZP-27538):
1. Removing main location is disallowed, but there is api to change main so this won't be a blocker right?
2. How do we handle initial language for versions? As it can be different one from version to version we will need to pick another one being initial version. 
3. We will also need to clarify what we do with drafts, they typically only contain one language, so either removing the translation involves removing the draft which I guess should throw.
4. with all the different cases of throws here I guess we should model this better to clearly tell what is wrong.